tests/Sitecore.AspNetCore.SDK.RenderingEngine.Tests/TagHelpers/Fields/ImageTagHelperFixture.cs

```diff
@@ -739,6 +739,456 @@ public void Process_RenderingChromesAreNotNull_ChromesAreOutput(
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert - <sc-img> generates HTML content, not attributes
+        string content = tagHelperOutput.Content.GetContent();
+        content.Should().Contain("srcset=");
+        content.Should().Contain("400w");
+        content.Should().Contain("200w");
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_SrcSetWithContentSDKBehavior_MatchesExpectedOutput(
+        ImageTagHelper sut,
+        TagHelperContext tagHelperContext,
+        TagHelperOutput tagHelperOutput)
+    {
+        // Arrange
+        tagHelperOutput.TagName = "img";
+        sut.For = GetModelExpression(new ImageField(_image));
+        sut.ImageParams = new { h = 1000 }; // Base parameters
+        sut.SrcSet = new object[] { new { h = 1000, w = 1000 }, new { mh = 250, mw = 250 } };
+
+        // Act
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert
+        tagHelperOutput.Attributes.Should().ContainSingle(a => a.Name == "srcset");
+        string srcSetValue = tagHelperOutput.Attributes["srcset"].Value.ToString()!;
+
+        // Should contain "1000w" (from w parameter taking priority over h)
+        srcSetValue.Should().Contain("1000w");
+
+        // Should contain "250w" (from mw parameter)
+        srcSetValue.Should().Contain("250w");
+
+        // Verify it contains the expected format: "url 1000w, url 250w"
+        string[] entries = srcSetValue.Split(", ");
+        entries.Should().HaveCount(2);
+        entries[0].Should().EndWith("1000w");
+        entries[1].Should().EndWith("250w");
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_SrcSetWithoutValidWidth_FiltersEntries(
+        ImageTagHelper sut,
+        TagHelperContext tagHelperContext,
+        TagHelperOutput tagHelperOutput)
+    {
+        // Arrange
+        tagHelperOutput.TagName = "img";
+        sut.For = GetModelExpression(new ImageField(_image));
+        sut.SrcSet = new object[] { new { h = 1000 }, new { mw = 250 }, new { quality = 80 } };
+
+        // Act
+        sut.Process(tagHelperContext, tagHelperOutput);
+
+        // Assert
+        tagHelperOutput.Attributes.Should().ContainSingle(a => a.Name == "srcset");
+        string srcSetValue = tagHelperOutput.Attributes["srcset"].Value.ToString()!;
+
+        // Should only contain the entry with mw parameter
+        srcSetValue.Should().Contain("250w");
+
+        // Should not contain entries without width parameters
+        string[] entries = srcSetValue.Split(", ");
+        entries.Should().HaveCount(1);
+    }
+
+    [Theory]
+    [AutoNSubstituteData]
+    public void Process_EditableImageWithSrcSet_MergesSrcSetIntoEditableMarkup(
+        ImageTagHelper sut,
+        TagHelperContext tagHelperContext,
+        TagHelperOutput tagHelperOutput)
+    {
+        // Arrange
+        Image image = new() { Src = "http://styleguide/-/media/img/sc_logo.png", Alt = "Sitecore Logo" };
```

Comment:
Why do we need to declare this image, can't we use the `_image` that the other tests use?